### PR TITLE
* Add characters:

### PR DESCRIPTION
**Pull request by jmcwilliams403**
- COMBINING TRIPLE DOT (`U+1AB4`).
- COMBINING TRIPLE ACUTE ACCENT (`U+1ACB`).

image

The triple dot was the harder one to get right. It's basically an equilateral triangle of diaeresis dots

### CODE REVIEW

File: font-src/glyphs/marks/above.ptl [hunk 1]
```diff
@@ -523,6 +535,23 @@ glyph-block Mark-Above : begin
 			flat (m2 - markStress) aboveMarkBot [widths markFine markFine]
 			curl (m2 + markExtend) aboveMarkTop [widths markStress markStress]
 
+	create-glyph 'tripleAcuteAbove' 0x1ACB : glyph-proc
```
Comment (be5invis) [inline font-src/glyphs/marks/above.ptl:538]: Please validate it in heavy weight. We may need thinner stroke width for it.

Comment (jmcwilliams403) [reply font-src/glyphs/marks/above.ptl:538]: Heaviest weight:
image

**Comment by jmcwilliams403** [comment]
after adjusting based on the 3/2 spacing:
image

File: font-src/glyphs/marks/above.ptl [hunk 2]
```diff
@@ -523,6 +535,26 @@ glyph-block Mark-Above : begin
...
+		set-width 0
+		include : StdAnchors.impl 'above' 0 1.5
+
+		local fine : markFine * 2 / 3
+		local stress : markStress * 2 / 3
```
Comment (be5invis) [inline font-src/glyphs/marks/above.ptl:543]: I think here we could use `[AdviceStroke]` with a larger crowdedness. Maybe this:

```
local fine : 0.5 * [AdviceStroke 6]
local stress : 1.2 * fine
```

Adjust this number "6" inside `[AdviceStroke]` to get a good visual under both thin, regular and heavy.

**Review by be5invis — CHANGES_REQUESTED** [review]

**Comment by jmcwilliams403** [comment]
Thin:
image
Regular:
image
Bold:
image
Heavy:
image

**Comment by jmcwilliams403** [comment]
For what it's worth, here's AdviceStroke 5

Thin:
image
Regular:
image
Heavy:
image

**Review by be5invis — APPROVED** [review]

**Pull request merged**